SU-HSDO/stanford_mrc
# Mrcd8 174 events news people

# READY FOR REVIEW

# Summary
Build visitor program/series page.

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Check out this branch
2. drush updb
3. drush en mrc_helper -y
4. Verify Distinguished Visitor entity reference field has been removed from visitor.
5. Verify Program Type vocabulary and terms have been deleted.
6. Verify MRC Event Series vocabulary has been created.
7. Verify MRC Event Series entity reference field is on event, news and visitor content types.
8. Verify blocks show/don't show up on term pages based on content.
9. We'll need to add subpatterns, image styles and see all links when styles are ready.

# Affected Projects or Products
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-174
